src/main/java/frc/robot/RobotContainer.java

```diff
@@ -101,9 +95,7 @@ private void configureButtonBindings() {
       OI.Operator.getUpBuddyBalanceButton().and(OI.Operator.getActivateBuddyBalanceButton()).onTrue(new InstantCommand(buddyBalanceSubsystem::retrieveBuddy, buddyBalanceSubsystem).unless(() -> !buddyBalanceSubsystem.getIsDeployed()));
     }
 
-    if (!Robot.isCompetition) {
-      OI.putControllerButtons();
-    }
+    OI.putControllerButtons();
```

Comment:
Do we want to put the controller buttons on shuffleboard in competition?

Reply:
yes, Adam said so